frontend/src/routes/manage/Video/index.tsx

```diff
@@ -278,18 +279,25 @@ const Row: React.FC<{ event: Events[number] }> = ({ event }) => {
     const link = `${PATH}/${keyOfId(event.id)}`;
     const { t, i18n } = useTranslation();
 
+    const deletionIsPending = Boolean(event.tobiraDeletionTimestamp);
+    const deletionFailed = Boolean(event.tobiraDeletionTimestamp
+        && Date.parse(event.tobiraDeletionTimestamp) + 60000 < Date.now());
```

Comment:
60s is too little time. Some institutions might increase the sync interval to >60s. This number depends on the exact message we use to communicate this fact. Maybe sth like "The deletion was request X time ago but still has not happened, so it might have failed."? And then I would still use like 10min as duration here.

In theory, we should check the sync interval and make sure this duration here is not shorter than the sync interval. Mhhh